lib/puppet/pops/types/string_converter.rb

```diff
@@ -385,6 +386,21 @@ def initialize
   # | s         | same as d
   # | p         | same as d
   #
+  # ### Binary (value)
+  #
+  # | Format    | Default formats
+  # | ------    | ---------------
+  # | s         | binary as unquoted characters
+  # | p         | 'Binary("<base64strict>")'
+  # | b         | '<base64>' - base64 string with newlines inserted
+  # | B         | '<base64strict>' - base64 strict string (without newlines inserted)
```

Comment:
It seems a bit inconsistent that '%s' maps to unquoted with newlines while '%B' maps to the `Binary#to_s()` method. Intuitively, I'd assume that '%s' would be the same as `#to_s()`.